Kernel/FileSystem/FATFS/Inode.cpp

```diff
@@ -319,9 +396,74 @@ ErrorOr<size_t> FATInode::write_bytes_locked(off_t offset, size_t size, UserOrKe
     return size;
 }
 
-ErrorOr<NonnullRefPtr<Inode>> FATInode::create_child(StringView, mode_t, dev_t, UserID, GroupID)
+ErrorOr<NonnullRefPtr<Inode>> FATInode::create_child(StringView name, mode_t mode, dev_t, UserID, GroupID)
```

Comment:
What happens when we fail during one of the `TRY`s in this function
should this attempt to do some cleanup steps, if yes maybe add a FIXME

Reply:
Added a FIXME, while I was at it I also added a FIXME in the allocate_entries function